gax/src/main/java/com/google/api/gax/paging/AbstractFixedSizeCollection.java

```diff
@@ -42,7 +41,6 @@
  *
  * <p>This is public only for technical reasons, for advanced usage.
```

Comment:
This comment seems incompatible with removing the `@BetaApi` annotation. Should anything marked with this description also be marked as `@Internal`? Alternatively, maybe we should remove that description?

Reply:
Good point - I think these comments should actually be removed in any classes with `@BetaApi` removed; for example, in this class there are functions that we expect end users to call (e.g. `getValues()`).